db-manager/common/utils/lib.js

```diff
@@ -34,8 +34,8 @@ function isXChainWallet(wallet) {
   return wallet.includes("/");
 }
 
-function taskRunner(task, db) {
-  return async (input) => await task(input, db);
+function taskRunner(task, db, extra) {
```

Comment:
`taskRunner(task, db, ...props)` can be used to support any number of extra params

```
function taskRunner(task, db, ...props) {
  if (props.length === 0) {
    return async (input) => await task(input, db);
  }
  return async (input) => await task(input, db, ...props);
}
```